### ISSUE TITLE
Restarting a previously completed function results in storage error

### ISSUE THREAD

**Issue by ethanroday** [Azure/azure-functions-durable-extension#276]
**Steps to reproduce:**

1. Create a very simple function like the one below (called `StatusTest`), which waits for a single event and then finishes.
2. Start an instance using `StartNewAsync()` from an HTTP triggered function, also below (called `Start`).
3. Send the `Finish` event through the HTTP API.
4. Start the instance again using `StartNewAsync()` from the HTTP triggered function.

**Expected result:**

After step 3, the function should be completed with the custom status `{ hello: "World" }`. The function should restart after step 4.

**Actual result:**

The function behaves as expected, but the second `StartNewAsync()` call results in an Azure storage error:

![image](https://user-images.githubusercontent.com/896670/39321992-5ce674b4-493d-11e8-859f-4a4843a3a50e.png)

Code for `Start`:

    public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, string instanceId, DurableOrchestrationClient client, TraceWriter log)
    {
        var existingInstance = await client.GetStatusAsync(instanceId);
        if (existingInstance == null || existingInstance.RuntimeStatus == OrchestrationRuntimeStatus.Completed) {
          await client.StartNewAsync("StatusTest", instanceId, null);
        }
        return req.CreateResponse(HttpStatusCode.OK);
    }

Code for `StatusTest`:

    public static async Task<bool> Run(DurableOrchestrationContext context, TraceWriter log)
    {
      context.SetCustomStatus(new { hello="World" });
      await context.WaitForExternalEvent<string>("Finish");
      return true;
    }

Also, the contents of my `local.settings.json`:

    {
      "IsEncrypted": false,
      "Values": {
        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
        "AzureWebJobsDashboard": "UseDevelopmentStorage=true"
      }
    }

**Environment Information:**
 - Local development on Windows 10 machine
 - Azure Storage Emulator 5.4.0.0
 - Durable Functions extension 1.3 RC
 - Functions runtime 2.0.11651.0

**Comment by cgillum** [comment]
Thanks for reporting this. I believe this may be a duplicate of https://github.com/Azure/azure-functions-durable-extension/issues/264. Can you try the workaround posted there to see if it resolves this issue?

**Comment by ethanroday** [comment]
That did the trick. Thanks!
